docs/content/guides/reverse-proxy.md

```diff
@@ -114,3 +114,25 @@ Here is an example config snippet:
   Include /etc/letsencrypt/options-ssl-apache.conf
 </VirtualHost>
 ```
+### Traefik
+
+In traefik we have to add the traefik labels and it is also very important: add our domain to "CMD_DOMAIN", "CMD_URL_ADDPORT=false", "CMD_PROTOCOL_USESSL=true".
```

Comment:
- We don't use "we" in the docs. We use "you".
- `add our domain to "CMD_DOMAIN", "CMD_URL_ADDPORT=false", "CMD_PROTOCOL_USESSL=true".` This sentence alone doesn't make much sense here. Where should it be added? To the hedgedoc config? If so, then it's not necessary because that is already covered by e.g. https://docs.hedgedoc.org/guides/reverse-proxy/#configure-asset-link-generation.